src/test/java/org/jfrog/buildinfo/integration/ArtifactoryPluginITest.java

```diff
@@ -66,11 +66,19 @@ public class ArtifactoryPluginITest extends TestCase {
             "/artifactory/libs-snapshot-local/org/example/maven-archetype-simple/1.0-SNAPSHOT/maven-archetype-simple-1.0-SNAPSHOT.pom"
     };
 
+    public void testMultiModuleInstallPluginVer2() throws Exception {
```

Comment:
Let's add a description for this test method, as well as the below `testMultiModuleInstallPluginVer3` method, specifically describing the difference between the two methods.